ortools/algorithms/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+ENABLE_LANGUAGE(CXX)
+# ENABLE_LANGUAGE(Python)
+SET(ROOT_PROJECT ${PROJECT_NAME})
+PROJECT(${PROJECT_NAME}_algorithms)
+
+FILE(GLOB ${PROJECT_NAME}_SOURCES "*.cc")
+LIST(REMOVE_ITEM ${PROJECT_NAME}_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/hungarian_test.cc")
```

Comment:
replace hungarian_test.cc by *_test.cc

Can you add one test rule per _test.cc?